q2_taxa/assets/barplot/src/toolbar.js

```diff
@@ -163,6 +163,13 @@ export function addSortByPicker(row, svg, data, dataMeta) {
       const sel = _appendSortByPicker(grp, svg, data, dataMeta);
       _sortBySelectOptions(sel, metaData, sortedKeysReverse, sortedKeysReverse[0]);
     });
+  grp.append('button').text('-')
```

Comment:
Can you make this a glyphicon? And [here](https://github.com/qiime2/q2-taxa/pull/46/files#diff-735c7160e4fd341ec6d00259c0e7b27eR157) too?